sklearn/linear_model/ridge.py

```diff
@@ -1586,14 +1586,17 @@ class RidgeCV(_BaseRidgeCV, RegressorMixin):
 
     Parameters
     ----------
-    alphas : numpy array of shape [n_alphas]
+    alphas : numpy array of shape [n_alphas], list or tuple
```

Comment:
I think you can revert the change here.

Most of scikit-learn docstring say "array" but will also work any iterable (e.g. list, tuples) that can be converted to arrays of the right shape.

Reply:
Thank you, done it.